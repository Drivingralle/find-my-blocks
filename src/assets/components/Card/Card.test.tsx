import React from "react";
import renderer from "react-test-renderer";
import { CardList, Card } from ".";

it("renders the CardList and Card", () => {
  const tree = renderer.create(
    <CardList title="Card List">
      <Card title="this is a title" postType="features" />
    </CardList>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it("tells that there is multiple usages in post", () => {
  const tree = renderer.create(
    <Card
      title="this is a title"
      postType="features"
      count={3} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a warning if isReusable", () => {
  const tree = renderer.create(
    <Card
      title="this is a title"
      postType="features"
      count={3}
      isReusable={ true } />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
