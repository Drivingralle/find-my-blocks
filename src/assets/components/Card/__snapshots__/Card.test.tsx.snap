// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Card 1`] = `
<div
  className="card"
>
  I am a card!!!
</div>
`;
